releases.yaml

```
@@ -8,7 +8,7 @@ latest:
   release_date: "2025年4月"
   eol: "2026年1月"
   past_eol_date: false
-  release_notes_url: "https://discourse.ubuntu.com/t/plucky-puffin-release-notes/48687"
+  release_notes_url: "https://discourse.ubuntu.com/t/plucky-puffin-release-notes"
```

```suggestion
  release_notes_url: "https://discourse.ubuntu.com/t/plucky-puffin-release-notes/48687"
```
Using the post id make this link more future proof in case the title is changed